src/test/java/uk/gov/hmcts/reform/bailcaseapi/domain/handlers/postsubmit/EndApplicationConfirmationTest.java

```diff
@@ -62,7 +62,7 @@ void should_set_confirmation_header_body() {
         assertEquals("# You have ended the application", response.getConfirmationHeader().get());
         assertThat(response.getConfirmationBody().get()).contains("#### What happens next\n\n");
         assertThat(response.getConfirmationBody().get())
-            .contains("A notification has been sent to all parties. No further action is required.");
+            .contains("A notification has been sent to all sendDirectionList. No further action is required.");
```

Comment:
The word "parties" got accidentally swapped for "sendDirectionList".

Reply:
I have changed sendDirectionList back to parties